src/path_tracking_pid_local_planner.cpp

```diff
@@ -413,14 +409,20 @@ uint8_t TrackingPidLocalPlanner::projectedCollisionCost()
   }
 
   // Fiddle the polygon into a marker message
-  std::vector<geometry_msgs::Point> collision_hull_points;
+  std::vector<tf2::Vector3> collision_hull_points;
   for (const geometry_msgs::Point point : collision_polygon) {
-    collision_hull_points.push_back(point);
+    tf2::Vector3 point_tf2;
+    tf2::fromMsg(point, point_tf2);
+    collision_hull_points.push_back(point_tf2);
```

Comment:
Consider using `tf2_convert()`.

Reply:
Well, `tf2_convert` calls `tf2::fromMsg` which does the conversion the other way around. Do we want to create two conversion functions for this? `from_msg` and `to_msg`?

Reply:
Oops. I do think that would be better but that doesn't need to be part of this PR.